homeassistant/helpers/entity_platform.py

```diff
@@ -282,21 +298,9 @@ def add_entities(self, new_entities, update_before_add=False):
 
         entity.hass = self.hass
         entity.platform = self
-
-        # Async entity
-        # PARALLEL_UPDATES == None: entity.parallel_updates = None
```

Comment:
This was the only docs we had. Are we adding it to the real docs now?

Reply:
I will do so.